tests/functional/test_data_completeness.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from botocore.session import Session
+from botocore.loaders import Loader
+from botocore.exceptions import DataNotFoundError
+
+
+class TestDataTypes(object):
```

Comment:
We should be evaluating this approach in the context of "Given this change is inconsistent with how all of the other botocore test generators are written, are there significant and immediately tangible benefits that make such inconsistencies worth it?"  And from what I'm seeing, my opinion is no, it's not worth it.

As you mention "should this file evolve to contain more test cases in the
future...".  So we don't actually need this right now.  When (*if*) we reach
such a scenario, then sure let's adapt and switch to this class based approach.

Until then, I prefer to keep it simple and consistent with the rest of botocore.